qiskit_ibm_runtime/runtime_program.py

```diff
@@ -101,6 +101,15 @@ def _format_common(schema: Dict) -> None:
                 )
             if "type" in schema:
                 formatted.append(" " * 4 + "Type: {}".format(str(schema["type"])))
+            if "input_allowed" in schema:
+                formatted.append(
+                    " " * 4 + "Input Allowed: {}".format(str(schema["input_allowed"]))
+                )
+            if "min_num_qubits" in schema:
+                formatted.append(
+                    " " * 4
+                    + "Minimum number qubits: {}".format(str(schema["min_num_qubits"]))
+                )
```

Comment:
description, type and properties don't exist in backend_requirements like the other three properties (interim_results, return_values and parameters). Also input_allowed and min_num_qubits don't exist in interim_results, return_values and parameters. So we shouldn't add this to the method named _format_common since it's not common. You can create another private method to format backend_requirements.